rbi/annotations/activesupport.rbi

```diff
@@ -46,3 +46,69 @@ class Object
   sig { returns(T::Boolean) }
   def present?; end
 end
+
+class Hash
+  sig { returns(T::Boolean) }
+  def extractable_options?; end
+end
+
+class Array
+  sig { params(position: Integer).returns(T.self_type) }
+  def from(position); end
+
+  sig { params(position: Integer).returns(T.self_type) }
+  def to(position); end
+
+  sig { params(elements: T.untyped).returns(T.self_type) }
+  def including(*elements); end
+
+  sig { params(elements: T.untyped).returns(T.self_type) }
+  def excluding(*elements); end
```

Comment:
I think we are missing:
```ruby
alias :without :excluding
```

Reply:
This (and for the `alias_method` suggestion) cause conflicts when typechecking:

```
sorbet/rbi/annotations/activesupport.rbi:95: Redefining the existing method Array#to_formatted_s as a method alias https://development.shopify.io/engineering/keytech/reference/sorbet/error_reference#5037
    95 |  alias_method :to_formatted_s, :to_fs
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    sorbet/rbi/gems/activesupport@7.1.0.alpha-2da863e0c380ef6bb9ddbd223a301554feb6a40e.rbi:14302: Previous definition
       14302 |  def to_formatted_s(format = T.unsafe(nil)); end
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I’ve tried regenerating the activesupport RBI but it still has a conflicting definition. 

Because of that, I've added "duplicated" method definitions here instead of using `alias*`